packages/webdriverio/src/commands/browser/touchAction.ts

```diff
@@ -59,6 +59,6 @@
 
 import { touchAction as touchActionCommand } from '../constant'
 
-export default function touchAction (...args) {
-    return touchActionCommand.apply(this, args)
+export default function touchAction (this: WebdriverIO.BrowserObject, ...args: WebdriverIO.TouchAction[]) {
```

Comment:
Shouldn't this be called `WebdriverIO.TouchActionArgs` instead of `WebdriverIO.TouchAction`?

Reply:
I guess it should be called `TouchActionOptions`, also something I would not like to change as part of this PR